src/views/Catalog.vue

```diff
@@ -5,25 +5,53 @@
         <ion-title>{{ $t("Catalog") }}</ion-title>
       </ion-toolbar>
     </ion-header>
-   
-    <ion-content :fullscreen="true">
-    
-    </ion-content>
+    <ion-content :fullscreen="true">    
+      <ion-searchbar></ion-searchbar>
+      <ion-row>
```

Comment:
Refer to markup of Clienteling app
https://github.com/hotwax/clienteling/blob/main/src/app/pages/home/home.html#L45

Reply:
Done